lib/files.js

```diff
@@ -122,6 +122,11 @@ module.exports.isDirectory = (file) => {
   return getStats(file).isDirectory();
 };
 
+module.exports.isVisibleFile = (path) => {
+  const pathParts = path.split('/');
```

Comment:
Apart from using files.splitPath (you'll need to export that function), it looks good :+1: